staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1187,6 +1222,23 @@ func (c *cacheWatcher) nextBookmarkTime(now time.Time) (time.Time, bool) {
 	return c.deadline.Add(-2 * time.Second), true
 }
 
+func getEventObject(object runtime.Object) runtime.Object {
+	if _, ok := object.(runtime.CacheableObject); ok {
+		return object
```

Comment:
a comment here would be helpful that this is safe to return without deep copying because we deep copied when we constructed the cacheable object

Reply:
done